website/docs/d/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+subcategory: "Oracle Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_exadata_infrastructure"
+description: |-
+  Gets information about an existing Exadata Infrastructure.
+---
+
+# Data Source: azurerm_oracle_exadata_infrastructure
+
+Use this data source to access information about an existing Exadata Infrastructure.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_exadata_infrastructure" "example" {
+  name                = "existing"
+  resource_group_name = "existing"
+}
+
+output "id" {
+  value = data.azurerm_oracle_exadata_infrastructure.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Exadata Infrastructure.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Exadata Infrastructure exists.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Exadata Infrastructure.
+
+* `activated_storage_count` - The requested number of additional storage servers activated for the Exadata infrastructure.
+
+* `additional_storage_count` - The requested number of additional storage servers for the Exadata infrastructure.
+
+* `available_storage_size_in_gbs` - The available storage can be allocated to the cloud Exadata infrastructure resource, in gigabytes (GB).
+
+* `compute_count` - The number of compute servers for the cloud Exadata infrastructure.
```

Comment:
```suggestion
* `compute_count` - The number of compute servers for the Cloud Exadata Infrastructure.
```

Reply:
Changed to Cloud Exadata Infrastructure.